datahub-web-react/src/app/homeV3/HomePageContent.tsx

```diff
@@ -11,7 +11,7 @@ const HomePageContent = () => {
         <ContentContainer>
             <CenteredContainer>
                 <ContentDiv>
-                    <EditHomePageSettingsButton />
+                    {/* <EditHomePageSettingsButton /> */}
```

Comment:
Minor: Maybe add comments aroud everything disabled in OSS to make the merge easier..